Makefile

```
@@ -94,30 +97,48 @@ CI_COMMIT_SHORT_SHA ?= $(shell git rev-parse --short HEAD 2>/dev/null || echo un
+        images-base images-core images-rest images-machine-validation \
+        images-boot-artifacts images-bfb
+
+images-validate:
+	$(call check-arches,$(NICO_ARCHES),NICO_ARCHES)
+
+images-all-validate: images-validate
+	$(call check-arches,$(BOOT_ARTIFACTS_ARCHES),BOOT_ARTIFACTS_ARCHES)
+	$(call check-arches,$(DPU_ARCHES),DPU_ARCHES)
+	$(require-nico-amd64)
+
+images: ## Build the deployable service stack (NICo Core + REST images)
```

checkmake (0.3.2)

[warning] Target body for "images" exceeds allowed length of 5 lines (7).

(maxbodylength)